src/pyflex/window_selector.py

```diff
@@ -911,12 +992,36 @@ def plot(self, filename=None):
         ax.spines['right'].set_color('none')
         ax.spines['left'].set_color('none')
         ax.spines['top'].set_color('none')
+        stepsize = 200
+        start, end = ax.get_xlim()
+        ax.xaxis.set_ticks(np.arange(start, end+1, stepsize))
```

Comment:
> I will update it and leave it to you to modify since you are more experienced...

Haha...I just like to play around. Yea you can leave that to me if you want.